addons/mail/models/ir_translation.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, _
+from odoo.exceptions import AccessError
+
+
+class IrTranslation(models.Model):
+    _inherit = 'ir.translation'
+
+    def _is_dynamic(self):
```

Comment:
it feels like we could avoid re-implementing this part, and delegate that check to the model itself, after browsing it with the appropriate context `lang`, in order to check the translated value